docroot/themes/humsci/su_humsci_theme/scss/components/molecules/_hover-menu.scss

```diff
@@ -364,6 +364,11 @@
         display: block;
       }
 
+      li ul li a,
+      li ul li ul li a {
```

Comment:
Do we NEED this level of depth? this seems incredibly specific and could cause complications later.